src/components/CheckList/CheckList.tsx

```diff
@@ -540,6 +514,22 @@ export const CheckList = ({ instance, onAddNewClick, checks, onCheckUpdate }: Pr
             setCurrentPage(1);
           }}
           labelFilters={labelFilters}
+          className={styles.marginRightSmall}
+        />
+        <AsyncMultiSelect
+          data-testid="probe-filter"
+          prefix="Probes"
+          onChange={(v) => {
```

Comment:
[nit] could this just be `onChange={setSelectedProbes}`?

Reply:
Will try!